Tests/Components Tests/Boleto/BoletoComponentTests.swift

```diff
@@ -124,18 +120,23 @@ class BoletoComponentTests: XCTestCase {
     }
     
     func testNoEmailSection() {
-        sut = BoletoComponent(paymentMethod: method,
-                              context: context,
-                              configuration: Dummy.getConfiguration(showEmailAddress: false))
         
-        let sutVC = sut.viewController
+        let context = Dummy.context(with: AnalyticsProviderMock())
```

Comment:
so the only meaningful change in `BoletoComponentTests.swift` is local `context` instead of setUp and Teardown?

Reply:
That's correct